mlir_graphblas/src/lowering-test/util.cpp

```diff
@@ -46,6 +46,22 @@ static FlatSymbolRefAttr getFunc(mlir::ModuleOp &mod, mlir::Location &loc, Strin
     return SymbolRefAttr::get(context, name);
 }
 
+static FlatSymbolRefAttr getVoidFunc(mlir::ModuleOp &mod, mlir::Location &loc, StringRef name,
+                                     TypeRange operands)
```

Comment:
I've found you don't need a separate `getVoidFunc()` if you change the signature for `getFunc()` to `
FlatSymbolRefAttr getFunc(mlir::ModuleOp &amp;mod, mlir::Location &amp;loc, StringRef name, TypeRange result,                                 TypeRange operands)`.  It seems to still work for non-void functions as-is.